app/models/manageiq/providers/automation_manager.rb

```diff
@@ -5,6 +5,8 @@ class ManageIQ::Providers::AutomationManager < ::ExtManagementSystem
   has_many :configured_systems,           :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_profiles,       :dependent => :destroy, :foreign_key => "manager_id"
   has_many :configuration_scripts,        :dependent => :destroy, :foreign_key => "manager_id"
+  has_many :credentials,                  :class_name => "ManageIQ::Providers::AutomationManager::Authentication",
```

Comment:
Didn't see Greg's list when typing up that comment. I am all for it and I've been raising that up from time to time (and stop short before people got annoyed 😅 )